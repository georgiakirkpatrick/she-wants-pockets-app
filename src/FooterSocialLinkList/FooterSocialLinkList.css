.Footer__social-link-list {
    background-color: rgb(236, 236, 236);
}