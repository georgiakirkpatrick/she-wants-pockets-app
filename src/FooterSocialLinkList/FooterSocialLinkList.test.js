import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import FooterSocialLinkList from './FooterSocialLinkList'

describe('FooterSocialLinkList', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<FooterSocialLinkList />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it ('render the UI as expected', () => {
        const tree = renderer
            .create(<FooterSocialLinkList />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})