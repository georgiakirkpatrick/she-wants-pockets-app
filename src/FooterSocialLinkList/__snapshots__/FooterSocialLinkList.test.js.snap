// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSocialLinkList render the UI as expected 1`] = `
<ul
  className="Footer__social-link-list"
/>
`;
